portfolio/src/main/webapp/script.js

```diff
@@ -39,8 +161,7 @@ function createCommentElement(comment) {
   deleteButtonElement.addEventListener("click", () => {
     deleteComment(comment);
 
-    // Remove the comment from the DOM.
-    commentElement.remove();
+    commentElement.remove(); // Remove the comment from the DOM.
   });
 
```

Comment:
I don't think we got to review the other functions below `deleteComment` and `deleteAllComments` but I would prefix those functions with `async` and also use `await` before the `fetch` calls.